tasks/configure-gpg.yml

```diff
@@ -0,0 +1,156 @@
+---
+
+# install jq
+
+- name: "Install jq package"
+  package: 
+    name: "jq"
+  become: yes
+
+# check when GPG Space already exists
+
+- name: "Count GPG Spaces"
+  shell: > 
+    curl -s -X GET 
+    -H "Authorization: ApiKey {{ archivematica_src_configure_ss_user }}:{{ archivematica_src_configure_ss_api_key }}" 
+    {{ archivematica_src_configure_ss_url }}/api/v2/space/?access_protocol=GPG 
+    | jq '.meta.total_count'
+  register: "gpg_spaces_count"
+  args:
+    warn: no
+    
+# add GPG space and locations when there is no GPG space already configured
+- name: "Add GPG space and locations"
+  block:
+
+    - name: "Set gpg binary name"
+      set_fact:
+        gpg_binary: "gpg1"
+      when: "ansible_distribution_version | version_compare('18.04', '>=')"
```

Comment:
Is this correct? We don't support ubuntu >= 18.04

Reply:
Yes, for future versions :P

Reply:
It's always good to think about the future ;)